microsetta_interface/templates/survey.jinja2

```diff
@@ -11,8 +11,8 @@
     <link rel="stylesheet" type="text/css" href="/static/vendor/vue-form-generator-2.3.4/vfg.css">
     <style type="text/css">
         img#bristol-chart {
-            height: auto;
-            width: 50%;
+            height: 50%;
```

Comment:
The height attribute isn't actually doing anything here, it can either be set to 'auto' or left out completely.

Also, setting width to 25% makes the image unreadable if I shrink my browser window. See attached screenshot. I think it would make more sense to set a max-width and min-width here to ensure it isn't ever excessively small or excessively large.

![bristol_screenshot_20220815](https://user-images.githubusercontent.com/83246693/184688706-80cb2f0f-db57-4e05-8306-01fcff91ed42.png)

Reply:
pushed new changes, please test